### ISSUE TITLE
I am authenticated by token and the video is private

### ISSUE THREAD

**Issue by lucasantoniooficial** [vimeo/laravel#78]
I am implementing vimeo through the library at Laravel and when I leave the video private only for me the video does not play, the error saying Private video and asks to login.

**Comment by Greenvold** [comment]
I think private videos are playable only on vimeo site after login. Try setting the video playable only on specific domains (localhost etc.) then embed the video.

**Comment by ikhattaab** [comment]
> I think private videos are playable only on vimeo site after login. Try setting the video playable only on specific domains (localhost etc.) then embed the video.

Not exactly, vimeo private videos are playable through the api if authenticated with access token.
